pkg/server/errorshandler/handler.go

```diff
@@ -89,3 +93,20 @@ func (handler *Handler) process(body []byte) ResponseMessage {
 
 	return ResponseMessage{200, false, "OK"}
 }
+
+// determineQueue - determine RabbitMQ route from catcherType
+func (handler *Handler) determineQueue(catcherType string) string {
+	if _, ok := handler.NonDefaultQueues[catcherType]; ok {
```

Comment:
I think, there should be a validation of the `catcherType`, because we can't trust an incoming message format.

Also, I think, it would be better if Event will contain the `catcherType` as a numeric constant instead of `errors/javascript`

Reply:
Legitimate queues are validated absolutely because of whitelist introduced by this request.
Concerning other `catcherType`s (which goes to the default queue), they can be checked by regex `^errors/[a-zA-Z]+$`. If you want I can add such regex with this request.